features/user.feature

```diff
@@ -152,6 +152,26 @@ Feature: Manage WordPress users
     Then STDERR should not be empty
     And the return code should be 1
 
+  Scenario: Trying to delete existing user with no roles from a subsite
+    Given a WP multisite install
+
+    When I run `wp user create bobjones bob@example.com --role=author --url=https://example.com --porcelain`
+    And save STDOUT as {BOB_ID}
+
+    When I run `wp user delete bobjones --yes`
+    Then STDOUT should be:
+      """
+      Success: Removed user {BOB_ID} from https://example.com.
```

Comment:
@danielbachhuber Is this `Then` more specific than necessary? I'm now noticing example that I could have copied

```
    When I run `wp user delete testsubscriber --yes`
    Then STDOUT should contain:
      """
      Success: Removed user
      """
```

Reply:
https://github.com/wp-cli/entity-command/blob/1f013e1b097a6af7a1ae85ea797f076486a7c749/features/user.feature#L345

Reply:
> Is this `Then` more specific than necessary? I'm now noticing example that I could have copied

@MannyAdumbire I don't have strong feelings one way or the other. A little less specific would be fine.

Reply:
cool! I made it less specific then for consistency with the other step.